src/main/java/bookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,15 @@
+package bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+import org.mapstruct.ReportingPolicy;
+
+@org.mapstruct.MapperConfig(
```

Comment:
You can use mapper config directly in BookMapper class in the @Mapper annotation